sktm/__init__.py

```diff
@@ -239,7 +249,8 @@ def check_pending(self):
                                 projid = int(patch.get("project_id"))
                             patches.append((pid, patch.get("name"), purl,
                                             baseurl, projid,
-                                            patch.get("date").replace(" ", "T")))
+                                            patch.get("date").replace(" ",
+                                                                      "T")))
```

Comment:
Could we perhaps wrap at the `.` operator here and in similar places?

I.e. write this instead:

    patches.append((pid, patch.get("name"), purl,
                    baseurl, projid,
                    patch.get("date").
                            replace(" ", "T")))

Reply:
Hi @veruu are you using a tool to re-style the code? Just to mention, there is a tool for this https://pypi.python.org/pypi/autopep8/. I tend to delegate the responsibility of styling in a tool to avoid the discussion (very often with myself hehe) how it'd be better.

Reply:
I'm aware of those tools, but the only one I use is the indentation plugin in vim. Based on my experience, those tools just blindly follow the pep8 (or other linters) rules and will screw up things you specifically did for readability. For example if you have a list of lists which you split into multiple lines (conforming to pep8), it will merge those lines together. So I opt to manually check the errors instead. Granted, I didn't do much for readability in this pull request (I just wanted to fix the errors with as little intervention as possible) and those tools might have improved since the last time I tried them so I might revisit this decision later.